package.json

```
@@ -3367,6 +3367,13 @@
 						"scope": "window",
 						"order": 20
 					},
+					"gitlens.integrations.experimental.gkdev.enabled": {
+						"type": "boolean",
+						"default": false,
+						"markdownDescription": "Specifies whether to enable authentication with GKDev",
+						"scope": "window",
+						"order": 30
+					},
```

The name and description of this setting make it seem as if disabling it would also disable the Jira integration, since it uses GKDev. I suggest making it specific to GitHub:
```suggestion
					"gitlens.experimental.cloudIntegrations.github.enabled": {
						"type": "boolean",
						"default": false,
						"markdownDescription": "Specifies whether to use the GitHub cloud integration when authenticating with GitHub",
						"scope": "window",
						"order": 30
					},
```

---

> Suggest a minor improvement on the setting name and description.

Fixed.

> but it seems that it should get merged before this, or otherwise it might not use the "manage cloud integrations" flow.

Merged.